mw-webapp/src/utils/DateUtils.ts

```diff
@@ -1,3 +1,6 @@
+const FIRST_INDEX = 0;
+const LAST_INDEX = 10;
```

Comment:
what does mean last index - it should be `array.length - 1`. Renaming required

Reply:
Rename to TENTH_INDEX. Is it ok?

Reply:
Rename to TENTH_INDEX. Is it ok?

Reply:
no

Reply:
Create tuple

Reply:
DATE_FROM_ISO_SUBSTRING_RANGE = [START_OF_ISO_SUBSTRING_RANGE, END_OF_ISO_SUBSTRING_RANGE]